softlayer/resource_softlayer_virtual_guest.go

```diff
@@ -615,6 +620,12 @@ func resourceSoftLayerVirtualGuestCreate(d *schema.ResourceData, meta interface{
 		return err
 	}
 
+	// Set notes
+	err = setNotes(id, d, meta)
+	if err != nil {
+		return err
+	}
+
```

Comment:
@renier As an example, if I POST the following request with `notes` set,
```
user:apiKey@api.softlayer.com/rest/v3/SoftLayer_Virtual_Guest/createObject
{
  "parameters": [
    {
      "hostname": "new-virtual-server",
      "domain": "test-iv.com",
      "startCpus": "1",
      "maxMemory": "2048",
      "localDiskFlag": "false",
      "hourlyBillingFlag": "true",
      "datacenter": {
        "name": "che01"
      },
      "notes": "Note given during provisioning",
      "operatingSystemReferenceCode": "DEBIAN_7_64"
    }
  ]
}
```

The created virtual server does not have the `notes` field populated.

I found something similar for subnets [here](http://stackoverflow.com/questions/38333724/adding-to-subnets-notes-field-after-ordering) wherein notes cannot be populated at the time of provisioning. It has to be done after provisioning is over.